### DISCUSSION TITLE
Enregistrement parcours

### DISCUSSION THREAD

**Discussion by Yann14280** [organicmaps/organicmaps#8652] [Ideas]
Bonjour
Existera t'il une possibilité d'enregistrement et d'export de ses parcours ( Gpx , Kml )?
Il me semble l'avoir aperçue sur une précedente version récente mais plus sur les dernières.
Merci pour votre travail.
Yann

**Comment by biodranik** [comment]
Bonjour, please follow this issue: #1785
